macros/L10nStatusOverview.ejs

```diff
@@ -8,41 +8,47 @@ var sectionColumnTitle = mdn.localString({
     "en-US": "Section",
     "de"   : "Bereich",
     "fr"   : "Catégorie",
-    "ru"   : "Раздел"
+    "ru"   : "Раздел",
+    "pt-BR": "Seção"
 });
 
 var pagesColumnTitle = mdn.localString({
     "en-US": "Pages",
     "de"   : "Seiten",
     "fr" : "Pages",
     "ru"   : "Страниц"
```

Comment:
Okay, thank you so much for letting me know, I'll be correcting as soon as possible.